frontend/package.json

```diff
@@ -10,6 +10,7 @@
     "babel-loader": "8.1.0",
     "bootstrap": "^5.0.0-beta3",
     "history": "^5.2.0",
+    "isomorphic-unfetch": "^3.1.0",
```

Comment:
It would be better not to take on another dependency which isomorphic-unfetch does the same thing that axios does, and axios is already in the code base.